### DISCUSSION TITLE
MoS2 example - raman.x does not working

### DISCUSSION THREAD

**Discussion by rodrigofisico** [nguyen-group/QERaman#20] [Q&A]
Hi everyone,

I'm trying to run the MoS2 example after successfully installing QERaman on the HPC. However, when I execute raman.x, I encounter the following screen (static):

![image](https://github.com/user-attachments/assets/cb7c93a3-bc9b-44dd-b6bc-6530e27655ee)

Additionally, I couldn't locate the MoS2.elph file in my working directory.

I'm unsure where the issue might lie. Any assistance you could offer would be greatly appreciated.

Thanks in advance,

Rodrigo

**Comment by nguyen-group** [comment]
Hi Rodrigo,

MoS2.elph should be in the same folder with ph.in. If you can not find it, ph_mat.x might not work correctly.
Could you check the ph.out to see if there are any errors? Or reinstall QERaman with QE version 7.2?

Nguyen

**Comment by rodrigofisico** [comment]
Dear Nguyen,

Thank you very much for your attention.
I am using the quantum espresso 7.3.1, I can not uninstall it on the HPC.
I am also running it again to check one more time, but all files are in the
same folder.
As soon as I get the results I will let you know.

Best Regards,

Rodrigo

**Comment by rodrigofisico** [comment]
Dear Nguyen,

I tryed different ways to run this example.
Before I thought it was less memory than necessary... I increase a lot it. And than, I selected only one node, to avoid problems with openMPI.
#SBATCH --nodes=1                        # Number of nodes to allocate for the job
#SBATCH --ntasks-per-node=25             # Number of tasks (processes) per node
#SBATCH --cpus-per-task=1                # Number of CPUs per task
#SBATCH --mem=80GB                      # Amount of memory allocated for the job
All options that I tryed before, falls at like the same.

I put the files attached.

Please let me know if you have any idea.

Best regards,

Rodrigo

The PH job do not finish...
[ph_47198534.out.txt](https://github.com/user-attachments/files/17109017/ph_47198534.out.txt)

      iter #   2 total cpu time :  1279.3 secs   av.it.:  17.8
      thresh= 6.910E-06 alpha_mix =  0.700 |ddv_scf|^2 =  1.294E-10

      iter #   3 total cpu time :  1992.6 secs   av.it.:  17.8
      thresh= 1.138E-06 alpha_mix =  0.700 |ddv_scf|^2 =  4.743E-12

      iter #   4 total cpu time :  2692.0 secs   av.it.:  17.3
      thresh= 2.178E-07 alpha_mix =  0.700 |ddv_scf|^2 =  2.976E-13

      iter #   5 total cpu time :  3401.9 secs   av.it.:  17.7
      thresh= 5.455E-08 alpha_mix =  0.700 |ddv_scf|^2 =  9.234E-14

      iter #   6 total cpu time :  3967.7 secs   av.it.:  13.7
      thresh= 3.039E-08 alpha_mix =  0.700 |ddv_scf|^2 =  2.481E-14

      iter #   7 total cpu time :  4575.9 secs   av.it.:  14.9
      thresh= 1.575E-08 alpha_mix =  0.700 |ddv_scf|^2 =  4.189E-17

      iter #   8 total cpu time :  5266.6 secs   av.it.:  16.6
      thresh= 6.472E-10 alpha_mix =  0.700 |ddv_scf|^2 =  2.245E-16

      iter #   9 total cpu time :  5926.9 secs   av.it.:  14.9
      thresh= 1.498E-09 alpha_mix =  0.700 |ddv_scf|^2 =  2.931E-19

The erro file appeared it:
[ph_47198534.err.txt](https://github.com/user-attachments/files/17109019/ph_47198534.err.txt)


Program received signal SIGSEGV: Segmentation fault - invalid memory reference.

Backtrace for this error:

Program received signal SIGSEGV: Segmentation fault - invalid memory reference.

Backtrace for this error:

Program received signal SIGSEGV: Segmentation fault - invalid memory reference.

Backtrace for this error:
